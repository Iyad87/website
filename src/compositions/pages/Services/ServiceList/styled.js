import styled from 'styled-components';
import { spacing } from 'styles/spacing';
import { textLight, textLighter } from 'styles/colours';

export const Illustration = styled.img`
  display: block;
  margin: ${spacing(5)} auto;
  max-width: 900px;
`;

export const Title = styled.h2`
  color: ${textLight};
  font-size: 2.25rem;
  font-weight: 500;
  margin: ${spacing(3)} 0;
`;

export const Support = styled.p`
  font-weight: 300;
  font-size: 1.5rem;
  color: ${textLighter};
  margin: ${spacing(3)} 0;
`;

export const Service = styled.li`
  margin-bottom: ${spacing(10)};
`;

export const List = styled.ul`
  margin: 0;
  padding: 0;
  list-style-type: none;
`;
