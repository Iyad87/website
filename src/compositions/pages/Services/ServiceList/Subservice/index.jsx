import React from 'react';
import PropTypes from 'prop-types';

import { Container, Title, Body } from './styled';

const Subservice = ({ children, name }) => (
  <Container>
    <Title>{name}</Title>
    <Body>{children}</Body>
  </Container>
);

Subservice.propTypes = {
  children: PropTypes.node,
  name: PropTypes.string.isRequired,
};

Subservice.defaultProps = {
  children: [],
};

export default Subservice;
