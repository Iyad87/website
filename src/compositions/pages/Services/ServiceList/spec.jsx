import React from 'react';
import { shallow } from 'enzyme';
import ServiceList from './index';

test('compositions/pages/Services/ServiceList snapshot', () => {
  const wrapper = shallow(<ServiceList />);

  expect(wrapper).toMatchSnapshot();
});
