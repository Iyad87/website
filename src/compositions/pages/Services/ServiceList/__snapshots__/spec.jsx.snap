// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compositions/pages/Services/ServiceList snapshot 1`] = `
<styled__List>
  <styled__Service
    id="approach"
  >
    <styled__Title>
      Our Approach
    </styled__Title>
    <styled__Support>
      We help you future proof your business though a new style of consulting, ground up. We connect and educate engineering, product and leadership to help you navigate digital transformation, grow your business and tackle core customer problems. We’re one of the most renowned technology companies in Europe.
    </styled__Support>
    <styled__Illustration
      src="http://placekitten.com/900/400"
    />
    <Subservice
      name="Engineering Consultancy"
    >
      We use our capability model to help you gauge your organisation’s current maturity. We help you identify any gaps and opportunities for improvement in line with your vision. We use our best practice models of discovery and delivery to drive success and ensure you have the right skills, processes and tools. What’s more we help you build the right framework to develop a long lasting engineering culture. We bring excellence, fast.
    </Subservice>
    <Subservice
      name="Product Consultancy"
    >
      We see a tremendous amount of innovation primarily because of organisations embracing to be product-driven. AirBnB now includes experiences and restaurants to their offering beyond just providing a place to stay. Investing to be a product-driven organisation will help you to future-proof and tackle areas of growth and innovation head-on.
      <br />
      <br />
      We can help you build the roadmap and ensure clear ways of working between the business and the delivery team. We can help you develop digital products in an Agile way of working.
    </Subservice>
    <Subservice
      name="Agile Organisations"
    >
      We use our capability model to help you gauge your organisation’s current maturity. We help you identify any gaps and opportunities for improvement in line with your vision. We use our best practice models of discovery and delivery to drive success and ensure you have the right skills, processes and tools. What’s more we help you build the right framework to develop a long lasting engineering culture. We bring excellence, fast.
    </Subservice>
  </styled__Service>
</styled__List>
`;
